src/components/Records/Record/GeneralInfo/Maintainers.vue

```diff
@@ -6,21 +6,29 @@
         v-if="getField('maintainers').length === 0"
         class="d-flex flex-wrap"
       >
-        <p class="ma-0 mr-2">
-          This record is in need of a maintainer
+        <p class="ma-0 mr-1">
+          This record is in need of a maintainer. If you are affiliated with this project,
         </p>
+
         <p
-          v-if="canClaim"
-          class="ma-0 mr-2"
+          v-if="!canClaim && !user().isLoggedIn"
+          class="ma-0 mr-1"
         >
-          If you are affiliated with this project,
+          log in and
         </p>
+        <router-link
+          v-if="!canClaim && !user().isLoggedIn"
+          to="/login"
```

Comment:
This should also be `@click="()=>{$emit('requestOwnership')}"` (as is "claim it now"), rather than a link to /login.

Reply:
![image](https://user-images.githubusercontent.com/25683981/126771374-728944cd-556e-4de6-b019-3d292521a5b3.png)

Allyson also in the comment mentioned turning login into a link.. please check if it's correct. tnx